rpc_jobs/rpc_openstack.yml

```diff
@@ -246,6 +292,110 @@
     jobs:
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
+- project:
+    name: "rpc-openstack-queens-aio-postmerge"
+    repo_name: "rpc-openstack"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branch: "queens"
+    jira_project_key: "RO"
+    # We have an MNAIO equivalent of this PM job, but we need to keep
+    # this for the push triggering for snapshot building.
+    CRON: "@monthly"
+    image:
+      - xenial_no_artifacts:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "7"
+    # This is the build that will be triggered by the push trigger job
+    trigger_build: 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+    jobs:
+      - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+      - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+- project:
+     name: "rpc-openstack-queens-rc-aio-postmerge"
+     repo_name: "rpc-openstack"
+     repo_url: "https://github.com/rcbops/rpc-openstack"
+     branch: "queens-rc"
+     jira_project_key: "RO"
+     # We have an MNAIO equivalent of this PM job, but we need to keep
+     # this for the push triggering for snapshot building.
+     CRON: "@monthly"
+     image:
+       - xenial_no_artifacts:
+           IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+     scenario:
+       - "swift"
+     action:
+       - deploy:
+           FLAVOR: "7"
+     # This is the build that will be triggered by the push trigger job
+     trigger_build: 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+     jobs:
+       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+       - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+- project:
+    name: "rpc-openstack-queens-mnaio-postmerge"
+    repo_name: "rpc-openstack"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branch: "queens"
+    jira_project_key: "RO"
+    image:
+      - xenial_mnaio_no_artifacts:
+          FLAVOR: "onmetal-io2"
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+          REGIONS: "IAD"
+          FALLBACK_REGIONS: "DFW"
+    scenario:
+      - ironic
+      - swift
+    action:
+      - system
```

Comment:
@johnduarte you all OK w/ these jobs appearing?

Reply:
Got feedback that ASC is happy with these appearing.

Reply:
👍